scout/index.md

```diff
@@ -66,7 +66,7 @@ For more information, see
 ### Artifactory integration
 
 Users of JFrog Artifactory, or JFrog Container Registry, can integrate Docker
-Scout to enable automatic analysis of images. For more information, see
+Scout to enable automatic analysis of images locally and in production. For more information, see
```

Comment:
Not sure if `in production` is accurate enough. We can analyze in any Artifactory instance which might be production but might equally be staging or test or whatever.

Reply:
```suggestion
Scout to enable automatic analysis of images locally and remotely. For more information, see
```

Sorry that should have been changed already.